app/views/shared/_equipment_landing.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="EquipoGrid">
```

Comment:
Is the shared/ directory the right place for this partial?

Reply:
I thought about that, my logic was that this partial might be used later for other places where the equipments are shown... What do you think?

Reply:
I think the shared directory is for things that appear in most or every view (i.e. flash messages, navigation elements, etc.). This partial should probaly go in views/equipment or something like that.